### DISCUSSION TITLE
Timeout in getRepeatableJobs() if Redis is down?

### DISCUSSION THREAD

**Discussion by OnnoGabriel** [OptimalBits/bull#2585] [Q&A] [answered]
Is there a way to define a timeout for asynchronous BullMQ methods?

My problem: `await queue.getRepeatableJobs()` returns all repeatable Jobs for a queue. However, when the Redis server is down, this function call lasts forever. There seems to be no timeout or other thrown error.

**Comment by manast** [comment, marked as answer]
https://docs.bullmq.io/patterns/failing-fast-when-redis-is-down

**Reply by OnnoGabriel** [reply]
Thanks a lot! I missed that part in the docs.
